homeworks/lesson2/app/src/components/course/list-course/list-course.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {ListGroup, ListGroupItem} from 'react-bootstrap';
+import './list-course.css';
+
+class Courses extends React.Component {
```

Comment:
Why the component name is Courses but file name as list-course?
Be more consistent in your code.